<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowProfileFind</class>
 <widget class="QWidget" name="ShowProfileFind">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>519</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>855</width>
    <height>519</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>855</width>
    <height>519</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #0082b6;</string>
  </property>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>480</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Chiudi</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="workingSkillsGroupBox">
   <property name="geometry">
    <rect>
     <x>479</x>
     <y>59</y>
     <width>351</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(78, 155, 209);
border-radius: 10px 10px 10px 10px;
border: 1px solid black;</string>
   </property>
   <property name="title">
    <string>Esperienze Lavorative</string>
   </property>
   <widget class="QTextBrowser" name="workingSkillsTextBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;
</string>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>30</y>
     <width>901</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt;
color:white;</string>
   </property>
   <property name="text">
    <string>Profilo utente cercato</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="constactsGroupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>351</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(78, 155, 209);
border-radius: 10px 10px 10px 10px;
border: 1px solid black;
color:white;</string>
   </property>
   <property name="title">
    <string>Contatti</string>
   </property>
   <widget class="QLabel" name="emailLabel_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color:white;</string>
    </property>
    <property name="text">
     <string>email:</string>
    </property>
   </widget>
   <widget class="QLabel" name="addressLabel_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color:white;</string>
    </property>
    <property name="text">
     <string>Indirizzo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="telephonLabel_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color:white;</string>
    </property>
    <property name="text">
     <string>Telefono:</string>
    </property>
   </widget>
   <widget class="QLabel" name="emailTextLabel_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
background-color:white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="AddressTextLabel_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
background-color:white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="telephonTextLabel_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
background-color:white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="qualificationGroupBox">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>200</y>
     <width>351</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(78, 155, 209);
border-radius: 10px 10px 10px 10px;
border: 1px solid black;</string>
   </property>
   <property name="title">
    <string>Esperienze Didattiche</string>
   </property>
   <widget class="QTextBrowser" name="qualificatioTextBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;
</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="knownLanguagesGroupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>351</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(78, 155, 209);
border-radius: 10px 10px 10px 10px;
border: 1px solid black;</string>
   </property>
   <property name="title">
    <string>Lingue Conosciute</string>
   </property>
   <widget class="QTextBrowser" name="knownLanguagesText_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;
</string>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>50</y>
     <width>20</width>
     <height>481</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="personalDataGroupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>351</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(78, 155, 209);
border-radius: 10px 10px 10px 10px;
border: 1px solid black;
color:white;</string>
   </property>
   <property name="title">
    <string>Dati Anagrafici</string>
   </property>
   <widget class="QLabel" name="nomeLabel_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>Nome:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cognomeLabel_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>Cognome:</string>
    </property>
   </widget>
   <widget class="QLabel" name="nomeTextLabel_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
background-color:white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cognomeTextLabel_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
background-color:white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="personalExperienceGroupBox">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>340</y>
     <width>351</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(78, 155, 209);
border-radius: 10px 10px 10px 10px;
border: 1px solid black;</string>
   </property>
   <property name="title">
    <string>Esperienze Personali</string>
   </property>
   <widget class="QTextBrowser" name="personalExperienceTextBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;
</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
